.github/workflows/integration_tests.yml

```diff
@@ -184,7 +184,8 @@ jobs:
         if: runner.os == 'macOS'
         run: sudo xcode-select -s /Applications/Xcode_${{ env.xcodeVersion }}.app/Contents/Developer
       - id: unity_setup
-        uses: firebase/firebase-unity-sdk/gha/unity@main
+        uses: firebase/firebase-unity-sdk/gha/unity@feature/retry-unity
```

Comment:
These should probably be using "./gha/unity"

Otherwise, it pulls from a different branch (usually main) which means changes to the scripts aren't tested until they are checked in.

Reply:
`./gha/unity` should work. Thanks for the suggestion.

I originally plan to have a different branch, but I am lazy😂.